samples/attestation/common/crypto.cpp

```diff
@@ -103,27 +97,39 @@ void Crypto::retrieve_public_key(uint8_t pem_public_key[512])
 }
 
 // Compute the sha256 hash of given data.
-int Crypto::Sha256(const uint8_t* data, size_t data_size, uint8_t sha256[32])
+bool Crypto::Sha256(const uint8_t* data, size_t data_size, uint8_t sha256[32])
 {
-    int ret = 0;
```

Comment:
Does this change the behaviour slightly? It may be beneficial to have a comment here
Before: `0` if a hash is successfully computed, and non-zero for any errors. Default `0`
After: `true` if a hash is successfully computed, and `false` for any errors. Default `false`

Reply:
The overall behavior should not change - if a single API call in this function fails, it should immediately go to exit still. Like above, the main reason I made this change is to make the code cleaner and more consistent, since other operations also return boolean.